api-specs/Gateway-EE/latest/kong-ee.yaml

```diff
@@ -12586,32 +12586,35 @@ paths:
                 properties:
                   data:
                     type: array
-                    properties:
-                      group:
-                        type: object
-                        properties:
-                          id:
-                            type: string
-                          name:
-                            type: string
-                          comment:
-                            type: string
-                          updated_at:
-                            type: timestamp
-                      rbac_role:
-                        type: object
-                        properties:
-                          id:
-                            type: string
-                          name:
-                            type: string
-                      workspace:
-                        type: object
-                        properties:
-                          id:
-                            type: string
-                  next:
-                    nullable: true
+                    items:
```

Comment:
Why is there `items` here? 
<img width="1146" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/22835296/fa47259a-ae95-40a4-8010-9e674684d811">